oneview/resource_fc_network.go

```diff
@@ -164,8 +158,13 @@ func resourceFCNetworkRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("created", fcNet.Created)
 	d.Set("modified", fcNet.Modified)
 	d.Set("etag", fcNet.ETAG)
-	d.Set("scopeuri", fcNet.ScopesUri.String())
-	d.Set("initial_scope_uris", fcNet.InitialScopeUris)
+	d.Set("scopesuri", fcNet.ScopesUri.String())
+	//d.Set("initial_scope_uris", fcNet.InitialScopeUris)
+	initialscopeuris := make([]string, 0, len(fcNet.InitialScopeUris))
+	for _, initialscopeuri := range fcNet.InitialScopeUris {
+		initialscopeuris = append(initialscopeuris, initialscopeuri.String())
+	}
+	d.Set("initial_scope_uris", initialscopeuris)
```

Comment:
during get we don't get initial_scope_uri so is it necessary

Reply:
done